internal/commands/user/inputs.go

```diff
@@ -93,7 +93,7 @@ func (i *usersInputs) ResolveUsers(ui terminal.UI, client realm.Client, app real
 	askErr := ui.AskOne(
 		&selectedUsers,
 		&survey.MultiSelect{
-			Message: "Which user(s) would you like to delete?",
+			Message: "Which user(s) would you like to select?",
```

Comment:
made this text more generic since revoke reuses the same function

Reply:
I'm debating whether this should be passed in as an arg, or keeping it generic. Thoughts?

Reply:
I think it would definitely be 'best' to be more descriptive. In terms or code maintainability and simplicity I like just using 'select' here personally. Though if anyone with a strong preference wants an arg for the message, I am not opposed to it

Reply:
we can be descriptive here... let's add an arg to `ResolveUsers` here in the form of `action string` (place it up front, before `ui`) and rename the function to `reoslveUsersFor` (is there still a reason this needs to be exported?)

that way, its usage ends up reading like:

```go
users, usersErr := cmd.inputs.resolveUsersFor("delete", ui, cmd.realmClient, app)
```